codex/purchasing/states/submitted.nim

```diff
@@ -27,7 +33,7 @@ method run*(state: PurchaseSubmitted, machine: Machine): Future[?State] {.async.
     await subscription.unsubscribe()
 
   proc withTimeout(future: Future[void]) {.async.} =
-    let expiry = request.expiry.truncate(int64)
+    let expiry = request.expiry.truncate(int64) + 1
```

Comment:
This is not a logging change?

Reply:
Right, I stumbled upon it and changed. Will move it to the #561 as it is more related...